src/registrar/beneficiary-details/beneficiary-details.component.ts

```
@@ -132,38 +135,42 @@ export class BeneficiaryDetailsComponent implements OnInit, DoCheck, OnDestroy {
+                ? ' ' + this.beneficiary.lastName
+                : '');
+            this.regDate = moment
+              .utc(this.beneficiary.createdDate)
+              .format('DD-MM-YYYY hh:mm A');
+          }
+        });
+      const benFlowID: any = param['beneficiaryRegID'];
+      this.beneficiaryDetailsService
+        .getBeneficiaryImage(benFlowID)
+        .subscribe((data: any) => {
+          if (data && data.benImage) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)